nilearn/datasets/atlas.py

```diff
@@ -1228,12 +1229,12 @@ def fetch_atlas_pauli_2017(version='prob', data_dir=None, verbose=1):
     Scientific Data, 5, 180063-13. http://doi.org/10.1038/sdata.2018.63``
     """
 
-    if version == 'prob':
+    if version == 'probabilistic':
         url_maps = 'https://osf.io/w8zq2/download'
-        filename = 'pauli_2017_labels.nii.gz'
-    elif version == 'labels':
+        filename = 'pauli_2017_probabilistic.nii.gz'
+    elif version == 'deterministic':
```

Comment:
Similarly, to preserve backward compatibility, make it:
```python
if version == 'deterministic' or version=='det':
```
Do not add the prob or det to the documentation.